UNATTENDED.md

```diff
@@ -0,0 +1,74 @@
+
+You can also manually create an `unattended.ini` configuration file in the root of the `eosimages` partition on a reformatter USB stick, following the documentation below. All fields are optional. Just creating an empty file will cause most screens to be skipped (if possible), though confirmation will be required to begin the reformatting process. This file should be UTF-8 text, without a [byte-order mark](https://en.wikipedia.org/wiki/Byte_order_mark), conforming to the `.ini`-style format defined in the [Desktop Entry Specification](https://specifications.freedesktop.org/desktop-entry-spec/latest/) and GLib's [Key-Value file parser](https://developer.gnome.org/glib/stable/glib-Key-value-file-parser.html) documentation.
+
+## Language
+
+By default, the reformatter user interface will be shown in _English (US)_. To present the reformatter in a different language, add a section like this to `unattended.ini`:
+
+```ini
+[EndlessOS]
+locale=pt_BR.utf8
+```
+
+In the example above, the reformatter will be displayed in _Brazilian Portuguese_. You may list the available locales on an Endless OS system by running the following in a Terminal window:
+
+```console
+$ locale -a | grep utf8
+```
+
+Note that this setting has _no effect_ on the default language for the _installed_ system.
+
+## Target Computer
+
+To reduce the possibility of accidentally booting an unattended installer on a system with valuable data, we recommend that the disk is pre-configured with target computers. On these systems, installation will commence _without any keypresses required_, after a 30-second countdown showing what action will be taken.
+
+These are specified in `unattended.ini` with option groups starting with `Computer`. Within each entry, `vendor` and `product` must both be specified, otherwise the entry is ignored. Here is an example targeting two computers:
+
+```ini
+[Computer 1]
+vendor=Asus
+product=X441SA
+
+[Computer 2]
+vendor=GIGABYTE
+product=GB-BXBT-2807
+```
+
+The correct values for `vendor` and `product` for a given system can be found in `/sys/class/dmi/id/sys_vendor` and `/sys/class/dmi/id/product_name` respectively.
+
+If no computer entries are provided, all computers will be reformatted, but explicit confirmation will be required before the process begins. If computers are specified, using the installer on a different computer will show a warning, and again require explicit confirmation to proceed.
+
+## Image
+
+If there is only one Endless OS image file on the USB stick, and only one disk present in the system (ignoring the one the installer is running from), no configuration is needed: that image will be written to that disk. Otherwise, add an option group beginning `Image` specifying the `filename` and/or `blockdevice` to use:
+
+```ini
+[Image 1]
+filename=eos-eos3.3-amd64-amd64.180115-104625.en.img.gz
+blockdevice=sd
```

Comment:
Nitpick: Typically, key names in the G world use hyphens to separate words. Could this be `block-device` instead?

Reply:
It could – I think it reads better too.